Algorithm.CSharp/BasicTemplateCryptoAlgorithm.cs

```diff
@@ -203,25 +203,25 @@ public override void OnEndOfAlgorithm()
         /// </summary>
         public Dictionary<string, string> ExpectedStatistics => new Dictionary<string, string>
         {
-            {"Total Trades", "10"},
-            {"Average Win", "0%"},
-            {"Average Loss", "-0.13%"},
-            {"Compounding Annual Return", "-99.979%"},
-            {"Drawdown", "3.500%"},
-            {"Expectancy", "-1"},
-            {"Net Profit", "-2.289%"},
-            {"Sharpe Ratio", "-11.341"},
-            {"Loss Rate", "100%"},
-            {"Win Rate", "0%"},
-            {"Profit-Loss Ratio", "0"},
-            {"Alpha", "-5.74"},
-            {"Beta", "1.931"},
-            {"Annual Standard Deviation", "0.254"},
-            {"Annual Variance", "0.065"},
-            {"Information Ratio", "-35.579"},
-            {"Tracking Error", "0.123"},
-            {"Treynor Ratio", "-1.494"},
-            {"Total Fees", "$85.33"}
+            {"Total Trades", "16"},
```

Comment:
We might need clarification on why the total trades count changed too?

Reply:
The algorithm uses indicators to determine the trades.

In turn, previous to the changes in [Algorithm/QCAlgorithm.Indicators.cs ](https://github.com/QuantConnect/Lean/pull/3993/files/81f201e7dfa954f2081393551a8a73c8afcd0ee2#diff-95fe47a1b5da9a59bc5c9cf89a905ca0R1744), Crypto indicators were updated with trades, now the indicators are updated from quotes.

Reply:
>now the indicators are updated from quotes.
I think crypto should also be updated with trades like equities. As we've seen quotes can vary so widely its almost random noise.  This would be reversing the default tick type right? It would also make it simpler for crypto users to create indicators?

Reply:
> This would be reversing the default tick type right?

Exactly.